examples/api-v3/updated_since/updated_since.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+// Useful resources:
+//
+// http://www.php.net/manual/en/function.json-decode.php
+// http://www.php.net/manual/en/language.types.array.php
+
+require './vendor/autoload.php';
+require_once '../../class/RestRequest.php';
+// require_once '../../class/JSON_support.php'; // Only required if using the PHP4 version
+
+# Application credential can be found at https://www.bookingsync.com/en/partners/applications
+# once you've created your first BookingSync application.
+$provider = new Bookingsync\OAuth2\Client\Provider\Bookingsync([
+    'clientId'          => 'XXXXXXXX',
+    'clientSecret'      => 'XXXXXXXX',
+    'redirectUri'       => 'https://localhost/updated_since/updated_since.php', // https is mandatory for BookingSync
+    'scopes'            => ['public'] // scopes required by your BookingSync application.
+]);
+
+session_start();
```

Comment:
No need to handle auth process here, should be done before in an other file / endpoint